tests/torch/test_hybrid_converter.py

```diff
@@ -229,3 +227,14 @@ def test_gpt2_hybrid_mlp_module_not_found():
     fake_module_name = "does_not_exist"
     with pytest.raises(ValueError, match=f"No module found for name {fake_module_name}"):
         HybridFHEModel(model, module_names=fake_module_name)
+
```

Comment:
do we somewhere have tests with brevitas models, since we say it works for both torch and brevitas?

Reply:
We have : test_hybrid_brevitas_qat_model